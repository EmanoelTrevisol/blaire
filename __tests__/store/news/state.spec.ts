import initialState from '@store/news/state';

describe('store/news/state.ts', () => {
  test('matches snapshot', () => {
    expect(initialState).toMatchSnapshot();
  });
});
