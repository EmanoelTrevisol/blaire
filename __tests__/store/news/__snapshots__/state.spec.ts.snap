// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store/news/state.ts matches snapshot 1`] = `
Object {
  "list": Array [],
}
`;
